ons_alpha/jinja2/templates/pages/topics/topic_page.html

```diff
@@ -1 +1,47 @@
 {% extends "templates/pages/index_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% block main %}
+    {% block summary %}
+        <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+            <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                <section id="summary">
+                    <h2>{{ ("Summary")}}</h2>
+                    <p>{{ page.summary }}</p>
+                </section>
+            </div>
+        </div>
+    {% endblock %}
+
+    {% block topic_page_nav %}
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+        <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+            <section id="topic_page_nav">
+                {% for key in subpage_dict.keys() %}
+                    <a href="#{{ key }}">{{ key }}</a>
+                {% endfor %}
+            </section>
+        </div>
+    </div>
+    {% endblock %}
+
+    {% block index %}
+        <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+            <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                {% if subpages %}
+                    <h2>{{"Index"}}</h2>
+                    {% for subpage_key in subpage_dict.keys() %}
+                        <h3 id="{{ subpage_key }}">{{ subpage_key }}</h3>
+                        {% for subpage_value in subpage_dict[subpage_key]  %}
```

Comment:
my nested loop logic
one header for each page type present 
and a list of all the child pages that are of the above page type
(this would then have associated pages (tagged) added to the list by page type)
I'm sorry I can't see any way of removing the second loop

Reply:
Ah damn, you're right - I'd misead!

However, it might be cleaner to use `.items` regardless:

```
{% for subpage_key, subpage_values in subpage_dict.items() %}
    {% for subpage_value in subpage_values %}
```